src/styles/index.ts

```diff
@@ -4819,6 +4819,11 @@ const styles = (theme: ThemeColors) =>
         textLineThrough: {
             textDecorationLine: 'line-through',
         },
+
+        searchFiltersTabItem: {
+            flexBasis: '33%',
+            minWidth: 200,
```

Comment:
the source of the problem is that `Text` inside `TabSelectorItem` is positioned `absolute`, so the parent tab does not stretch to fit the text.
because of that I need basis so the button/tab doesn't grow to 100% and also `mindWidth` so that the tab doesn't get too small - otherwise it would alway be 33% which may be too small and look bad.

`minWidth` is not crucial but I think it makes it look better.

Reply:
Yes I'm 100% aware of that, however I wanted to use existing component (that is also used when starting a new chat and starting new expense) instead of adding new code.

However Im not sure whether the scope of this task (Search related) should also contain fixing this existing button. Perhaps we should create a separate task for taking a look at this `TabSelectorItem`? I would prefer not to affect other views when working on this PR.

Reply:
I agree with @Kicu that we should keep the changes in this PR contained to the Search page. If we need to make style changes to `TabSelectorItem`, let's create a separate issue that focuses on that and takes into account all views using `TabSelectorItem`.